index.html

```diff
@@ -3,46 +3,47 @@
   <head>
     <title>Cristian Moran</title>
     <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lobster">
   </head>
-  <body>
-    <div>
-      <!-- Landing -->
-      <h1>
-        <div>
-          <img
-            src="file:///C:/Users/cmora/OneDrive/Pictures/Welcome.png"
-            width="600"
-          />
+  <body class="body">
+    <!-- Background img -->
+    <div class="underwater"></div>
+    <!-- Landing -->
+      <div class="iframe">
+        <iframe style="border-radius:12px" src="https://open.spotify.com/embed/track/5407a4uk868QuLmcvB58gh?utm_source=generator" width="30%" height="100" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture" loading="lazy">
+        </iframe>
+        </div>
+        <div class="welcome">
+          <h1>Welcome</h1>
         </div>
-      </h1>
-      <div>
+      <div class="direction-buttons">
         <nav>
           <ul>
-            <button id="About"><a href="#aboutHeading">About</a></button>
-            <button id="Experience">
-              <a href="#experienceHeading">Experience</a>
-            </button>
-            <button id="Connect"><a href="#connectHeading">Connect</a></button>
+            <li id="tag"><a href="#aboutHeading">About</a></li>
+            <li id="tag"><a href="#experienceHeading">Experience</a></li>
+            <li id="tag"><a href="#skills">Skills</a></li>
+            <li id="tag"><a href="#connectHeading">Connect</a></li>
```

Comment:
you can only have one item per page with the same `id`, so this is a good example of when you'd want to do `class="tag"` for all of these instead